File: app/src/main/java/mega/privacy/android/app/fragments/managerFragments/cu/CuTitleViewHolder.java
package mega.privacy.android.app.fragments.managerFragments.cu;

import android.util.Pair;

import androidx.core.text.HtmlCompat;

import mega.privacy.android.app.R;
import mega.privacy.android.app.databinding.ItemCameraUploadsTitleBinding;

import static mega.privacy.android.app.utils.LogUtil.logWarning;
import static mega.privacy.android.app.utils.StringResourcesUtils.getString;

class CuTitleViewHolder extends CuViewHolder {

    private final ItemCameraUploadsTitleBinding mBinding;

    public CuTitleViewHolder(ItemCameraUploadsTitleBinding binding) {
        super(binding.getRoot());

        mBinding = binding;
    }

    @Override
    protected void bind(CuNode node) {
        Pair<String, String> date = node.getHeaderDate();
        String dateText = getString(R.string.bold_month_year_date, date.first, date.second);

        try {
            dateText = dateText.replace("[B]", "<font face=\"sans-serif-medium\">")
                    .replace("[/B]", "</font>");
        } catch (Exception e) {
            logWarning("Exception formatting text.", e);
        }

        mBinding.headerText.setText(HtmlCompat.fromHtml(dateText, HtmlCompat.FROM_HTML_MODE_LEGACY));
    }

    @Override protected boolean handleClick() {
        return false;
    }
}

File: app/src/main/java/mega/privacy/android/app/fragments/managerFragments/cu/CuTitleViewHolder.kt
package mega.privacy.android.app.fragments.managerFragments.cu

import mega.privacy.android.app.R
import mega.privacy.android.app.databinding.ItemCameraUploadsTitleBinding
import mega.privacy.android.app.utils.LogUtil
import mega.privacy.android.app.utils.StringResourcesUtils
import mega.privacy.android.app.utils.StringUtils.toSpannedHtmlText
import mega.privacy.android.app.utils.TextUtil

class CuTitleViewHolder(
    private val mBinding: ItemCameraUploadsTitleBinding
) : CuGridViewHolder(mBinding.root) {

    override fun bind(node: CuNode) {
        val date = node.headerDate
        var dateText =
            if (TextUtil.isTextEmpty(date!!.second)) "[B]" + date.first + "[/B]" else StringResourcesUtils.getString(
                R.string.cu_month_year_date,
                date.first,
                date.second
            )
        try {
            dateText = dateText.replace("[B]", "<font face=\"sans-serif-medium\">")
                .replace("[/B]", "</font>")
        } catch (e: Exception) {
            LogUtil.logWarning("Exception formatting text.", e)
        }

        mBinding.headerText.text = dateText.toSpannedHtmlText()
    }

    override fun handleClick() = false
}